docs/src/develop/components/ArticleDoc.js

```diff
@@ -32,6 +32,8 @@ var ArticleDoc = React.createClass({
         <section>
           <h2>Options</h2>
           <dl>
+            <dt><code>primary     true|false</code></dt>
+            <dd>Whether it should be treated as main content or not.</dd>
```

Comment:
Can we specify that this is commonly used for Accessibility as the `Main Content` of the page?